src/adera/ShipResources.h

```diff
@@ -141,7 +140,7 @@ struct ShipResourceType
 
 struct ShipResource
 {
-    osp::DependRes<ShipResourceType> m_type;
+    //osp::DependRes<ShipResourceType> m_type;
```

Comment:
imho, just erase the line

Reply:
Something will eventually replace this, though I don't fully remember how the ShipResource stuff works. I intentionally left this not deleted so its original intention can still be worked out.